project/npda/models/patient.py

```diff
@@ -113,24 +124,31 @@ def __str__(self) -> str:
     def get_absolute_url(self):
         return reverse("patient-detail", kwargs={"pk": self.pk})
 
+    def get_todays_date(self) -> date:
```

Comment:
Could we avoid this mocking entirely by passing in the date as a parameter directly from the tests?

Reply:
`age_days` was put in to calculate age in years by dividing by 365.25y

Reply:
Yes but do you know where it was supposed to be used? At the moment I don't think it's used anywhere